algolia/query_suggestions/client_config.go

```diff
@@ -0,0 +1,68 @@
+package query_suggestions
+
+import (
+	"fmt"
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/call"
+	"net/http"
+)
+
+type IndexConfig struct {
+	// Index name to target.
+	IndexName string `json:"indexName"`
+	// List of source indices used to generate a Query Suggestions index.
+	SourceIndices []SourceIndex `json:"sourceIndices"`
+	// De-duplicate singular and plural suggestions.
+	Languages []string `json:"languages,omitempty"`
+	// List of words and patterns to exclude from the Query Suggestions index.
+	Exclude []string `json:"exclude,omitempty"`
+}
+
+type SourceIndex struct {
+	// Source index name.
+	IndexName string `json:"indexName"`
+	// List of analytics tags to filter the popular searches per tag.
+	AnalyticsTags []string `json:"analyticsTags,omitempty"`
+	// List of facets to define as categories for the query suggestions
+	Facets []map[string]interface{}
+	// Minimum number of hits (e.g., matching records in the source index) to generate a suggestions.
+	MinHits *int `json:"minHits"`
+	// Minimum number of required letters for a suggestion to remain.
+	MinLetters *int `json:"minLetters"`
+	// List of facet attributes used to generate Query Suggestions. The resulting suggestions are every combination of the facets in the nested list
+	Generate [][]string `json:"generate,omitempty"`
+	// TODO: Figure out what type we should use for external.
+	// List of external indices to use to generate custom Query Suggestions.
+	// External []string `json:"external,omitempty"`
```

Comment:
```suggestion
	External []string `json:"external,omitempty"`
```
This is the list of indices names, so the `[]string` type is a good one.

Reply:
Thank you for your confirmation! Fixed to use `[]string`